### PR TITLE
LOL 404

### PR DESCRIPTION
**Pull request by commuterjoy**
lol

### CODE REVIEW

File: client/main.scss [hunk 1]
```diff
@@ -28,15 +26,16 @@ $output-critical: true !default;
 
 	.error-page__title {
 		@include oTypographySansBold(xl);
+		margin-bottom: 20px;
```
Comment (bjfletcher) [inline client/main.scss]: replace:

`@include oTypography...(...);`

with:

`@include oTypography...(..., $load-progressively: true);`

for a progressive font experience

File: client/main.scss [hunk 2]
```diff
@@ -46,11 +45,33 @@ $output-critical: true !default;
...
 	}
-}
 
-.looking-for {
-	@include oTypographySans(m);
-	padding-top: 40px;
+	.not-found__list {
+		@include oGridRespondTo('L') {
+			column-count: 2;
+			-webkit-column-count: 2;
```
Comment (bjfletcher) [inline client/main.scss]: use `autoprefixer`

File: client/main.scss [hunk 3]
```diff
@@ -46,11 +45,33 @@ $output-critical: true !default;
...
+	.not-found__definition {
+		margin: 0 0 20px 0;
+	}
+
+	.not-found__citation {
+		display: inline-block;
+		width: 16px;
+		height: 16px;
+		border-radius: 10px;
+		background-color: rgba(224, 138, 172, 0.54);
```
Comment (bjfletcher) [inline client/main.scss]: `rgba` won't work on IE8, add a `background-color` fallback above this... or not worry about IE8 users

Comment (commuterjoy) [inline client/main.scss]: ok, moved to a normal rgb

**Pull request merged**